### DISCUSSION TITLE
Is sentry-react-native v5.3.0 at feature parity with v4.+.+?

### DISCUSSION THREAD

**Discussion by fabiendem** [getsentry/sentry-react-native#2943] [Q&A]
Hey,

Question is in the title.
We're currently running the latest v4, and are contemplating the v5.+ releases.
What's not clear to us is if we can make the move, does v5 has all the same features than v4, or is still catching up (seems like a big revamp)?

Cheers!
Fabien

**Comment by krystofwoldrich** [comment]
Hi,
thanks for the question.

You can go ahead and use v5+ releases. They have all the features from v4.x.x plus all new features are added only to v5. 

We have a migration document that should make the upgrade as smooth as possible.

https://docs.sentry.io/platforms/react-native/migration/#from-4x-to-5x

If you would run into something please let us know.

**Reply by fabiendem** [reply]
Thanks 👍
